### ISSUE TITLE
Support for request decoration

### ISSUE THREAD

**Issue by a2l007** [allegro/turnilo#117]
First of all I'm glad to know there is an active open source project that is on top of Swiv. Planning to give this a whirl soon. Does turnilo provide an option for request decoration, something similar to what Pivot does here: https://docs.imply.io/on-premise/special-ui-features/advanced-usage#druid-authentication ?

**Comment by mkuthan** [comment]
We did not touch request decoration so it should work, but we do not use this feature so there is chance that something has been broken ;)

**Comment by piter75** [comment]
We are open to community contributions.

Regarding the drag and drop functionality problem. Do you by any chance observe it in Firefox browser? If it's not Firefox then what browser (with version) do you use?

I have just found a regression in drag and drop handling code in Firefox introduced in one of the recent versions.

**Comment by a2l007** [comment]
Yeah I was testing it on Firefox 59. Seems to work fine on Chrome though.

**Comment by piter75** [comment]
I have filed another issue (#118) to track the drag and drop bug.
